tools/moth.py

```diff
@@ -325,18 +316,58 @@ def secrets(self):
 
 class Category:
     def __init__(self, path, seed):
+        """Represents a new category of puzzles.  
+        Gathers the point values for all the puzzles in the category.          
+          1. It pulls the points from the POINTS attribute of path/category.py, if it exists.
+          2. Any path/<integer_dir> is counted as a puzzle as well.
+        When puzzles are generated, they are first built using the completely optional
+        puzzle directory, then passed to the category.make function if it is in 
+        category.POINTS.
+        :param str path: Path to the category directory.
+        :param float seed: The random seed for this categories random number generator."""
+        
         self.path = path
         self.seed = seed
+        
+        if 'category.py' in os.listdir(path):
+            # Good Lord this is forking dangerous.
```

Comment:
Better, but let's just leave this out of code that we're going to share with vendors and the rest of the world :)